### PR TITLE
DOCSP-40797: Specify a query

### PR DESCRIPTION
**Pull request by norareidy**
# Pull Request Info

[PR Reviewing Guidelines](https://github.com/mongodb/docs-java/blob/master/REVIEWING.md)

JIRA - https://jira.mongodb.org/browse/DOCSP-40797
Staging - https://preview-mongodbnorareidy.gatsbyjs.io/cpp-driver/DOCSP-40797-query/read/specify-a-query/

## Self-Review Checklist

- [ ] Is this free of any warnings or errors in the RST?
- [x] Did you run a spell-check?
- [x] Did you run a grammar-check?
- [x] Are all the links working?
- [ ] Are the [facets and meta keywords](https://wiki.corp.mongodb.com/display/DE/Docs+Taxonomy) accurate?

### CODE REVIEW

File: source/read/specify-a-query.txt [hunk 1]
```diff
@@ -0,0 +1,267 @@
...
+
+The following code example shows how to create a database and collection, then
+insert the sample documents into your collection:
+
+.. literalinclude:: /includes/read/specify-queries.cpp
+   :start-after: start-setup
+   :end-before: end-setup
+   :language: cpp
+   :dedent:
+   :copyable:
```
[the remark this answers is not in the document]
Comment (norareidy) [reply source/read/specify-a-query.txt:53]: Yeah it's a little repetitive - I removed the json code block

File: source/includes/read/specify-queries.cpp [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+    // start-find-array
+    auto cursor = collection.find(make_document(kvp("type", make_document(kvp("$size", 2)))));
+    for(auto&& doc : cursor) {
+        std::cout << bsoncxx::to_json(doc) << std::endl;
+    }
+    // end-find-array
+
+    // Retrieves and prints documents that have a "color" field
+    // start-find-element
+    auto cursor = collection.find(make_document(kvp("color", make_document(kvp("$exists", "true")))));
```
Comment (kevinAlbs) [inline source/includes/read/specify-queries.cpp:75]:
```suggestion
    auto cursor = collection.find(make_document(kvp("color", make_document(kvp("$exists", true)))));
```

Though a string appears to work, [documentation](https://www.mongodb.com/docs/manual/reference/operator/query/exists/) indicates a boolean is expected.

**Review by kevinAlbs — APPROVED** [review]
LGTM with minor changes.

**Review by laurelxiang — APPROVED** [review]
LGTM

**Pull request merged**